slash/utils/traceback_utils.py

```diff
@@ -142,14 +144,17 @@ def to_string(self, include_vars=False):
 
 class DistilledFrame(object):
 
-    def __init__(self, frame, lineno=None):
+    def __init__(self, frame, lineno=None, repr_blacklisted_types=None):
         super(DistilledFrame, self).__init__()
         self.python_frame = frame
         self.filename = os.path.abspath(frame.f_code.co_filename)
         if lineno is None:
             lineno = frame.f_lineno
         self.lineno = lineno
         self.func_name = frame.f_code.co_name
+        if repr_blacklisted_types is None:
+            repr_blacklisted_types = tuple(config.root.log.repr_blacklisted_types)
+        self._repr_blacklisted_types = repr_blacklisted_types
```

Comment:
I'm not sure this should be a property on the class instance - all we do is use it from the constructor